src/renderer/routes/test_sequencer_panel/components/TestSequencerInfo.tsx

```diff
@@ -54,49 +47,69 @@ const TestSequencerView = () => {
-        {/*     disabled={IS_CLOUD_DEMO} */}
-        {/*   /> */}
-        {/*   <Input */}
-        {/*     className={INPUT_FIELD_STYLE} */}
-        {/*     value={testRunTag} */}
-        {/*     onChange={(e) => setTestRunTag(e.target.value)} */}
-        {/*     placeholder="Test Run Tag (optional)" */}
-        {/*     disabled={IS_CLOUD_DEMO} */}
-        {/*   /> */}
-        {/*   <Button>New Test</Button> */}
-        {/* </div> */}
-
-        <div className="w-5/6">
-          <DataTable />
-          <SummaryTable />
-        </div>
+        <div className="flex overflow-y-auto">
+          <div
+            className="ml-auto mr-auto h-3/5 flex-grow flex-col overflow-y-auto"
+            style={{ height: "calc(100vh - 260px)" }}
+          >
+            <SummaryTable />
+            <DataTable />
+          </div>
 
-        {/* Test Flow Control buttons */}
-        <div className="flex flex-row space-x-5">
-          <LockableButton onClick={handleClickImportTest}>
-            + Import test
-          </LockableButton>
-          <LockableButton onClick={handleClickSaveTestSet}>
-            Save Test Set
-          </LockableButton>
-          <LockableButton onClick={handleClickImportTestTest}>
-            Import Test Set
-          </LockableButton>
-          {/* <LockableButton>Export test</LockableButton> */}
-          <LockableButton onClick={handleClickRunTest}>Run Test</LockableButton>
+          <div>
+            <div className="top-0 h-full flex-none overflow-y-auto pl-5">
+              <CloudPanel />
+              <div className="mt-5 rounded-xl rounded-xl border border border-gray-300 border-gray-300 p-4 py-4 dark:border-gray-800">
+                <div className="flex flex-col">
+                  <h2 className="mb-2 pt-3 text-center text-lg font-bold text-accent1 ">
+                    {" "}
+                    Control Panel{" "}
+                  </h2>
+                  <LockableButton
+                    className="mt-4 w-full"
+                    variant="outline"
+                    onClick={handleClickImportTest}
+                  >
+                    {" "}
+                    Import Python Tests{" "}
+                  </LockableButton>
+                  <LockableButton
+                    className="mt-4 w-full"
+                    variant="outline"
+                    onClick={handleClickImportTestTest}
+                  >
+                    {" "}
+                    Import Test Set{" "}
+                  </LockableButton>
+                  <LockableButton
+                    className="mt-4 w-full"
+                    variant="outline"
+                    onClick={handleClickSaveTestSet}
+                  >
+                    {" "}
+                    Save Test Set{" "}
+                  </LockableButton>
+                  <LockableButton
+                    data-cy="btn-play"
+                    data-testid="btn-play"
```

Comment:
There is a button with same id already present [here](https://github.com/flojoy-ai/studio/blob/main/src/renderer/routes/flow_chart/views/ControlBar/FlowControlButtons.tsx#L98)